tests/conftest.py

```diff
@@ -50,6 +50,7 @@ async def app():
     clean_database(settings)
     app = create_app()
     await app.router.startup()
+    settings.first_install = False
```

Comment:
isn't the default value `False`?

Reply:
yes bit it will run the admin ui setup inside the tests which makes it beeing set to True.
this is needed here else the Redirects happen inside the tests